dashboards/etcd/etcd.json

```diff
@@ -19,7 +19,7 @@
               "id": "viz.markdown"
             },
             "rawConfiguration": {
-              "text": "![New Relic logo](https://i.imgur.com/zQS8meU.png) \n- - -\n### Overview\n`etcd` is a strongly consistent, distributed key-value store that provides a reliable way to store data that needs to be accessed by a distributed system or cluster of machines. It gracefully handles leader elections during network partitions and can tolerate machine failure, even in the leader node.\n\n### About this Dashboard\n\nThis dashboard is based on standard `etcd` Prometheus metrics ingested using Prometheus [remote write](https://newrelic.com/instant-observability/prometheus-remote-write/7dbd7355-c380-4211-bd32-fed9a65351af).  Most widgets utilize an `instance` facet to identify `etcd` instances in the cluster.  \n\n**Note**:  if you're running a managed Kubernetes cluster (e.g. EKS, AKS, GKE), then the full set of `etcd` Prometheus metrics will not be available in these platforms.\n\n"
+              "text": "![New Relic logo](https://i.imgur.com/zQS8meU.png) \n- - -\n### Overview\n`etcd` is a strongly consistent, distributed key-value store that provides a reliable way to store data that needs to be accessed by a distributed system or cluster of machines. It gracefully handles leader elections during network partitions and can tolerate machine failure, even in the leader node.\n\n### About this Dashboard\n\nThis dashboard is based on standard `etcd` Prometheus metrics ingested using Prometheus [remote write](https://newrelic.com/instant-observability/prometheus-remote-write/7dbd7355-c380-4211-bd32-fed9a65351af).  Most widgets utilize an `instance` facet to identify `etcd` instances in the cluster.  \n\n**Note**:  if you're running a managed Kubernetes cluster (e.g. EKS, AKS, GKE), then the full set of `etcd` Prometheus metrics will not be available in these platforms and DB Total Size (MB) widget is supported by etcd 3.5 and later versions. \n\n"
```

Comment:
based on notes below for the query, you may want to update this to account for older versions too